client/src/pages/FloorPlan.js

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 
-const Desk = ({ desk }) => {
+const Desk = ({ desk, bookingForThisDesk }) => {
 	return (
 		<>
 			<g transform={`rotate(${desk.r} 75 75)`}>
 				<rect
 					style={{
-						fill: "#3F94D5",
+						fill: bookingForThisDesk ? "red" : "green",
```

Comment:
can you move this style to a stylesheet and use className here?

Reply:
looks good with a wee change to reflect the new colour scheme.

Reply:
@gregdyke Yes, I will.
@TheCheekyGit let's do it at the relevant PR. "Style-Modal"